src/compiler/compile/nodes/Element.ts

```diff
@@ -451,6 +451,26 @@ export default class Element extends Node {
 			const href_attribute = attribute_map.get('href') || attribute_map.get('xlink:href');
 			const id_attribute = attribute_map.get('id');
 			const name_attribute = attribute_map.get('name');
+			const target_attribute = attribute_map.get('target');
+
+			if (target_attribute && target_attribute.get_static_value() === '_blank' && href_attribute) {
+				const href_static_value = href_attribute.get_static_value() ? href_attribute.get_static_value().toLowerCase() : null;
+
+				if (href_static_value === null || href_static_value.startsWith('http') || href_static_value.startsWith('//')) {
```

Comment:
make sure it starts with `http://` or `https://`

checking if it starts with `http` may have false positive with `<a href="httpa/b/c/d" />`

Reply:
Hello, thx for your feedback @tanhauhau.

It's fixed. I took this opportunity to add other tests, such as capital letters etc. (https vs HTTPS)

Feel free to add another feedback if needed !

See ya !

Reply:
I let you resolve the conversation if this fix is ok for you @tanhauhau ?